benchmark/src/main/java/com/amazon/opendistroforelasticsearch/sql/benchmark/BenchmarkService.java

```diff
@@ -41,23 +41,29 @@
  * Class to run benchmarking, includes entry-point for benchmarking.
  */
 public class BenchmarkService {
+
   private List<String> types;
   private String outputFile;
   private String benchmarkPath;
   private String dataDirectoryPath;
   private List<Double> scaleFactors;
-  private String systemPassword;
+  public static String systemPassword;
```

Comment:
How come these are static?

Reply:
needed user input for one db but function definition has to be same for all hence used static var so I can use directly

Reply:
I would instead use a context object with key/value pairs, and pull out the specific keys representing the information you need.

Reply:
will address this in separate PR